tests/dom/identity/test_canonical_auth_session.py

```
@@ -40,25 +42,25 @@ def test_DOM_IDEN_006__student_login_verifies_user_pin_and_resolves_through_clai
+        student.user.last_active_class_id = None
+        db.session.flush()
     response = student_login(client, username=student.username, pin=student.pin)
 
     assert response.status_code == 302
     assert "/student/select-class-context" in response.headers["Location"]
 
 
-def test_DOM_IDEN_006__student_login_no_valid_class_seats_hard_fails(client, monkeypatch):
+def test_DOM_IDEN_006__student_login_no_active_class_redirects_to_selector(client, monkeypatch):
     monkeypatch.setattr("app.routes.student.verify_turnstile_token", lambda *_args, **_kwargs: True)
     classroom, student = initialize_as_student("chemistry_p1", client, client.application)
```

Ruff (0.16.1)

[warning] Unpacked variable `classroom` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)